app/kumactl/cmd/install/testdata/install-control-plane.defaults.golden.yaml

```diff
@@ -4384,6 +4407,8 @@ spec:
               value: kong-docker-kuma-docker.bintray.io/kuma-dp:0.0.1
             - name: KUMA_INJECTOR_INIT_CONTAINER_IMAGE
               value: kong-docker-kuma-docker.bintray.io/kuma-init:0.0.1
+            - name: KUMA_INJECTOR_INIT_CONTAINER_DISABLED
```

Comment:
Can we have `KUMA_INJECTOR_INIT_CONTAINER_ENABLED` instead of DISABLED? I'd like this to be consistent with `KUMA_KUBERNETES_CNI_ENABLED`

Reply:
I wanted that omitting of that value in `yaml` configuration had the same behavior as it were before changes:
```yaml
initContainer:
  image: kuma-init:latest
```
Otherwise, we have to explicitly write:
```yaml
initContainer:
  enable: true
  image: kuma-init:latest
```
But that's not a big deal to change to `KUMA_INJECTOR_INIT_CONTAINER_ENABLED`

Reply:
I understand but I'd go with this anyways. We control the deployment of injector